online/src/main/scala/ai/chronon/online/FetcherBase.scala

```diff
@@ -295,6 +295,12 @@ class FetcherBase(kvStore: KVStore,
     isCachingFlagEnabled
   }
 
+  // If the flag is set, we check whether the entity is in the active entity list saved on k-v store
+  def isEntityValidityCheckEnabled: Boolean = {
+    Option(flagStore)
+      .exists(_.isSet("enable_entity_validity_check", Map.empty[String, String].asJava))
```

Comment:
we don't have flag store implemented for Airbnb yet. How do we check otherwise?

Reply:
Hi @pengyu-hou the flagStore can be added in airbnb kv-store like this PR https://git.musta.ch/airbnb/treehouse/pull/773151/files